index.html

```diff
@@ -0,0 +1,120 @@
+        <img class="photos__img" src="images/photos/6-compressor.jpg" alt="">
+      </div>
+      <p class="gallery__caption">Faena House / <span class="highlighted">Miami Beach, FL 33140</span></p>
+      <button class="button gallery__button">View all</button>
+    </div>
+  </section>
+  <section class="section">
+    <h2 class="heading section__heading">Consultation<span class="heading__crown"></span> </h2>
+    <div class="consultation">
+      <div class="consultation__text">
+        <h3 class="consultation__Heading">How long have<br>you do yourself a gift?</h3>
+        <p>The procedural change mezzo forte starts izoritmichesky pickup at these moments stop LA Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.</p>
+      </div>
+      <form class="form consultation__form">
+        <label><input type="text" name="name" placeholder="Name" class="input form__input"></label>
+        <label><input type="text" name="phone" placeholder="Phone" class="input form__input"></label>
+        <label><input type="email" name="email" placeholder="Email" class="input form__input"></label>
+        <button class="button form__button">Consultation</button>
+      </form>
+    </div>
+  </section>
+  <section class="section">
+    <h2 class="heading section__heading">Testimonials<span class="heading__crown"></span> </h2>
+    <div class="testimonials section__content">
+      <div class="person">
+        <img class="person__photo" src="images/people/1.jpg" alt="Tim Cook">
+        <h3 class="person__name">Tim Cook</h3>
+        <p class="person__position">CEO</p>
+        <p class="person__text">Nice work, Certificates National Association of Realtors (USA).</p>
+      </div>
+      <div class="person">
+        <img class="person__photo" src="images/people/2.jpg" alt="Katherine Adams">
+        <h3 class="person__name">Katherine Adams</h3>
+        <p class="person__position">Senior Vice President</p>
+        <p class="person__text">My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart. I’m all alone and blissfully happy in the local edge.</p>
+      </div>
+      <div class="person">
+        <img class="person__photo" src="images/people/3.jpg" alt="Jonathan Ive">
+        <h3 class="person__name">Jonathan Ive</h3>
+        <p class="person__position">Chief Design Officer</p>
+        <p class="person__text">When my lovely valley of steam rises and half-day sun is above an impermeable.</p>
+      </div>
+    </div>
+  </section>
+  <section class="section">
+    <h2 class="heading section__heading">Contact Us<span class="heading__crown"></span> </h2>
+    <div class="contact">
+      <div class="contact__text">
+        <p><a class="highlighted" href="mailto:hello@miami.com">Hello@miami.com</a><br>+1 234 555-55-55</p>
```

Comment:
will be cool if you add number to `a` use `href='tel'`
```suggestion
        <p>
          <a class="highlighted" href="mailto:hello@miami.com">
            Hello@miami.com
          </a>
          <br>+1 234 555-55-55
        </p>
```